docs/writing-docs/docs-page.md

```diff
@@ -1,205 +1,188 @@
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'common/button-story-disable-docspage-component.js.mdx',
+    'common/storybook-preview-auto-docs-custom-template.js.mdx',
+    'common/storybook-preview-auto-docs-template.mdx.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-### With MDX documentation
+<div class="aside">
 
-Write your documentation in MDX and update the `docs.page` [parameter](../writing-stories/parameters.md) to display it. The `id` of reference follows the pattern: `group-subgroup-...--name`, where the `groups` and `subgroups` are defined as according to the [Grouping Documentation](https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#grouping).
+💡 If you only need to override the documentation page for a single component, we recommend creating an MDX file and referencing it directly via the `<Meta of={} />` Doc Block.
+
+</div>
+
+## Setup custom documentation
+
+Storybook provides support for MDX 2, an open standard markup language, combining two other ones: Markdown, which is used for formatting text, and JSX, which is used for rendering dynamic elements on a page. To enable custom documentation for your stories with this format, start by updating your Storybook configuration file (i.e., `.storybook/main.js|ts|cjs`).
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'common/custom-docs-page.mdx.mdx',
-    'common/button-story-docspage-with-mdx.js.mdx',
+    'common/storybook-auto-docs-main-mdx-config.js.mdx',
+    'common/storybook-auto-docs-main-mdx-config.ts.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-### With a custom component
-
-Storybook's UI is built using React. If you want to include a custom component to display documentation, you'll need to update your environment to allow React components to be correctly transpiled.
-
-For example, with Angular start by adding a `babel.config.js` file at the root of the project with the following content:
-
-```js
-// babel.config.js
-
-module.exports = function (api) {
-  process.env.NODE_ENV === 'development' ? api.cache(false) : api.cache(true);
-  const presets = [
-    [
-      '@babel/preset-env',
-      {
-        targets: {
-          node: 'current',
-        },
-      },
-    ],
-    '@babel/preset-typescript',
-    '@babel/preset-react',
-  ];
-  const plugins = [];
-  return {
-    presets,
-    plugins,
-  };
-};
-```
-
-Then, update your `tsconfig.json` to include the following:
-
-```json
-{
-  "compilerOptions": {
-    ....
-    "allowJs": true,
-    "jsx": "react-jsx",
-  },
-}
-```
-
-Finally write your custom React component and update the `docs.page` [parameter](../writing-stories/parameters.md) to render the custom documentation.
+Create an MDX file for your component in the same directory as your stories and components to add your custom documentation. The file should match your component's name (e.g., `Button.mdx`).
```

Comment:
This sort of implies the name needs to be the same, rather than just that being a suggestion. What *needs* to happen is referencing the CSF file in a `<Meta of={}/>`